src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/monitoring-list/monitoring-list.component.html

```diff
@@ -0,0 +1,38 @@
+<tabset #tabs>
+  <tab id="active-alerts"
+       heading="Active Alerts"
+       (selectTab)="setFragment($event)"
+       i18n-heading>
+    <cd-active-alert-list *ngIf="isAlertmanagerConfigured"></cd-active-alert-list>
+    <cd-alert-panel type="warning"
+                    i18n
+                    *ngIf="!isAlertmanagerConfigured">To see all active Prometheus alerts, please
+      provide the URL to the API of Prometheus' Alertmanager as described in the
+  <a href="{{docsUrl}}"
+     target="_blank">documentation</a>.</cd-alert-panel>
+  </tab>
+  <tab id="all-alerts"
+       heading="All Alerts"
+       (selectTab)="setFragment($event)"
+       i18n-heading>
+    <cd-rules-list *ngIf="isPrometheusConfigured"
+                   [data]="prometheusAlertService.rules"></cd-rules-list>
+    <cd-alert-panel type="warning"
+                    *ngIf="!isPrometheusConfigured">To see all Prometheus configured alerts, please provide the URL to
+      the API of Prometheus as described in the
```

Comment:
This is used in other messages, too.
```suggestion
      the API of the Prometheus' Alertmanager as described in the
```

Reply:
Yes, but it would be incorrect to use it here. The other messages require to use the URL to the API of the Alertmanager, this one requires the URL to the API of Prometheus.